cfpb/cfgov-refresh
# Make search button visible on nonresponsive pages

Search button in header is not visible on pages that use the `base_nonresponsive` template (Find a housing counselor, Paying for college)

## Additions

- Override default button text color and letter spacing on `.m-global-search_trigger` in nemo stylesheet

## Testing

- Check out branch
- Run `gulp scripts:ondemand:nonresponsive` to generate header js for nonresponsive template
- Navigate to [find a housing counselor page](http://localhost:8000/find-a-housing-counselor/)
- Verify that search button is visible in header & functions correctly

## Review

- @cfpb/cfgov-frontends 

## Screenshots

### Search button hidden on find a housing counselor on cf.gov
<img width="1240" alt="screen shot 2016-09-08 at 10 15 50 am" src="https://cloud.githubusercontent.com/assets/778171/18359160/72acff9e-75ad-11e6-8a31-ed2d3b31dab6.png">

### Search button visible
<img width="1241" alt="screen shot 2016-09-08 at 10 16 14 am" src="https://cloud.githubusercontent.com/assets/778171/18359164/79a1cd34-75ad-11e6-92a6-b4e388484645.png">

## Checklist
